crates/chia-protocol/fuzz/fuzz_targets/streamable.rs

```diff
@@ -7,16 +7,13 @@ use sha2::{Digest, Sha256};
 
 pub fn test_streamable<T: Streamable + std::fmt::Debug + PartialEq>(obj: &T) {
     let bytes = obj.to_bytes().unwrap();
-    let obj2 = match T::from_bytes(&bytes) {
-        Err(_) => {
-            panic!(
-                "failed to parse input: {}, from object: {:?}",
-                hex::encode(bytes),
-                &obj
-            );
-        }
-        Ok(o) => o,
-    };
+    let obj2 = T::from_bytes(&bytes).unwrap_or_else(|_| {
+        panic!(
+            "failed to parse input: {}, from object: {:?}",
+            hex::encode(&bytes),
+            &obj
+        )
+    });
```

Comment:
this looks like it would be slightly simpler as:
```suggestion
    let Ok(obj2) = T::from_bytes(&bytes) else {
        panic!(
            "failed to parse input: {}, from object: {:?}",
            hex::encode(&bytes),
            &obj
        )
    };
```